bench/bench_csharp_preview.yaml

```diff
@@ -73,10 +77,10 @@ environments:
     compiler: dotnet
     version: 7
     compiler_version_command: dotnet --version
-    docker: mcr.microsoft.com/dotnet/sdk:7.0
+    docker: mcr.microsoft.com/dotnet/sdk:8.0-preview
     # docker_volumns:
     #   - /tmp/.nuget/packages:/root/.nuget/packages
     include: dotnet_preview
-    build: dotnet publish -c Release -r linux-x64 -f net7 --self-contained true -p:PublishSingleFile=true -o out # -p:PublishReadyToRun=true
+    build: dotnet publish -c Release -r linux-x64 -f net8 --self-contained true -p:PublishSingleFile=true -o out # -p:PublishReadyToRun=true
```

Comment:
we could target net7 just as well, but bench\include\dotnet_preview\app.csproj targets net8 so I had to change one or the other.